docker/Dockerfile

```diff
@@ -1,11 +1,13 @@
 FROM debian:bookworm as builder
```

Comment:
```suggestion
FROM debian:bookworm AS builder
```

Otherwise `docker build` throws the following warning:

```
WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
```